src/main/java/com/owncloud/android/ui/adapter/ActivityListAdapter.java

```diff
@@ -369,7 +368,8 @@ public void updateDrawState(@NonNull TextPaint ds) {
                     }
                 }, idx1, idx2, 0);
                 ssb.setSpan(new StyleSpan(android.graphics.Typeface.BOLD), idx1, idx2, 0);
-                ssb.setSpan(new ForegroundColorSpan(Color.BLACK), idx1, idx2, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+                ssb.setSpan(new ForegroundColorSpan(context.getResources().getColor(R.color.textColor)),
+                            idx1, idx2, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Comment:
for such methods with many parameters, we agreed to have a line break after each parameter, so pleas:

```suggestion
                            ssb.setSpan(new ForegroundColorSpan(context.getResources().getColor(R.color.textColor)),
                            idx1, 
                            idx2, 
                            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Reply:
> for such methods with many parameters, we agreed to have a line break after each parameter, so pleas:

Where can I find information on this and general code formatting? I don't see anything in the contribution document but I'll admit I haven't looked through the developer docs.